frontend/talentsearch/src/js/components/diversityEquityInclusion/dialogs/DialogActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { SaveIcon } from "@heroicons/react/solid";
+import { Button } from "@common/components";
+
+interface DialogActionsProps {
+  onDismiss: () => void;
+}
+
+const DialogActions: React.FC<DialogActionsProps> = ({ onDismiss }) => {
+  const intl = useIntl();
+  return (
+    <div
+      data-h2-display="b(flex)"
+      data-h2-flex-direction="b(column) m(row)"
```

Comment:
Seems like there's enough width here to show the buttons side by side.
![image](https://user-images.githubusercontent.com/8978655/172463728-d516fed9-290c-41c0-97d2-650d9f51aa7b.png)